src/Symfony/Component/Process/Process.php

```diff
@@ -1123,12 +1114,14 @@ private function captureExitCode()
     /**
      * Closes process resource, closes file handles, sets the exitcode.
      *
+     * @param bool $wait Whether to wait for the end of the process or not.
+     *
      * @return int The exitcode
      */
-    private function close()
+    private function close($wait)
```

Comment:
I'd just keep the `$this->stop(0);` in destructor and don't add the `$wait` parameter here. I would not feel good leaving some open resources